docs/additional-configuration.adoc

```diff
@@ -302,7 +302,9 @@ spec:
           securityContext:
             runAsUser: 1234
 ----
-Note that the pod-overrides field does not allow configuring `containers`, `initContainers`, and `volumes`. In order to configure these elements, use existing DevWorkspace fields (container components, volume components)
+Note that the pod-overrides field does not allow configuring `containers` and `initContainers`. In order to configure these elements, use the existing DevWorkspace `spec.template.components` field.
```

Comment:
Potential suggestion:

"Note that the pod-overrides field does not allow configuring **the** `containers` and `initContainers` **fields from the pod spec**".